cmd/runner/main.go

```diff
@@ -48,6 +48,11 @@ func main() {
 	cfg := makeRunnerConfig()
 	fmt.Println(cfg)
 
+	cfg, err := cfg.Validate()
```

Comment:
Actually we don't need to retrieve a new config from `Validate`, we could just do:
```go
if err := cfg.Validate(); err != nil { ... }
```

Reply:
You are right, it is not supposed to change anything in the configuration so it does not need to return it. I have changed it!